matching-service/models/user-orm.js

```diff
@@ -0,0 +1,61 @@
+import User from './user.js';
+import Sequelize, { DataTypes, Op } from 'sequelize';
+import sequelize from './repository.js';
+
+export async function createUser(id, name, difficulty) {
+    sequelize.sync().then((result) => {
+        User.create({
+            id: id,
+            name: name,
+            difficulty: difficulty
+        }) 
+    });
+}
+
+export async function numUsers() {
+    User.findAll().then((result) => {
+        console.log(result);
```

Comment:
There should not be console logs at the user-orm level. Please delete all logs from this file (only use for debugging purposes, then delete)